### PR TITLE
Updating workflows/scRNAseq/pseudobulk-worflow-decoupler-edger from 0.1 to 0.2

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following workflow: **workflows/scRNAseq/pseudobulk-worflow-decoupler-edger**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/ebi-gxa/decoupler_pseudobulk/decoupler_pseudobulk/1.4.0+galaxy5` should be updated to `toolshed.g2.bx.psu.edu/repos/ebi-gxa/decoupler_pseudobulk/decoupler_pseudobulk/1.4.0+galaxy8`

The workflow release number has been updated from 0.1 to 0.2.

If you want to skip this change, close this PR without deleting the branch. It will be reopened if another change is detected.
Any commit from another author than 'planemo-autoupdate' will prevent more auto-updates.
To ignore manual changes and allow autoupdates, delete the branch.

### CODE REVIEW

File: workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Changelog
 
+## [0.2] 2024-11-18
```
Comment (mvdbeek) [inline workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/CHANGELOG.md:3]:
```suggestion
## [0.1.1] 2024-11-18
```

File: workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/pseudo-bulk_edgeR.ga [hunk 2]
```diff
@@ -21,7 +21,7 @@
     ],
     "format-version": "0.1",
     "license": "CC-BY-4.0",
-    "release": "0.1",
+    "release": "0.2",
```
Comment (mvdbeek) [inline workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/pseudo-bulk_edgeR.ga:24]:
```suggestion
    "release": "0.1.1",
```

File: workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
+
```
Comment (mvdbeek) [inline workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/CHANGELOG.md:4]:
```suggestion
### Added
- Extended attribution to credit [Persist-SEQ](https://persist-seq.org/) as the original authors of the workflow that this workflow is based on.

```

**Review by mvdbeek — APPROVED** [review]

**Comment by mvdbeek** [comment]
@dianichj @pcm32 does this still look good ?

**Comment by pcm32** [comment]
Those changes in the decoupler tool introduced some additional features, but, everything should be backwards compatible. Will the workflow tests run?

**Comment by mvdbeek** [comment]
They did for the previous commit: https://github.com/galaxyproject/iwc/actions/runs/11886441070

**Comment by pcm32** [comment]
Tests seem to have passed now, so all good.

**Pull request merged**